internal/mode/static/state/graph/nginxproxy_test.go

```diff
@@ -71,6 +71,9 @@ func TestGetNginxProxy(t *testing.T) {
 					ObjectMeta: metav1.ObjectMeta{
 						Name: "np2",
 					},
+					Spec: ngfAPI.NginxProxySpec{
+						IPFamily: helpers.GetPointer(ngfAPI.Dual),
```

Comment:
Can we add a test with something other than Dual here to make sure that we only set IPFamily to Dual if it's nil?

Reply:
So the `NginxProxySpec` is nil 

```
nps: map[types.NamespacedName]*ngfAPI.NginxProxy{
				{Name: "np1"}: {
					ObjectMeta: metav1.ObjectMeta{
						Name: "np1",
					},
				},
				{Name: "np2"}: {
					ObjectMeta: metav1.ObjectMeta{
						Name: "np2",
					},
				},
			},
```

the returned output, is with `Dual`